<launch>
  <arg
    name="model" />
  <param
    name="robot_description"
    textfile="$(find uav_simulator)/urdf/Avatar.urdf" />
  <node
    name="rviz"
    pkg="rviz"
    type="rviz"
    args="-d $(find uav_simulator)/config/show.rviz" />
  <node pkg="tf2_ros" type="static_transform_publisher" name="son2" args="0 0 0 1.57 0 0 /base_link /laser_scan" output="screen" />

  <node pkg="uav_simulator" type="simulator" name="simulator" />
</launch>
